### DISCUSSION TITLE
How can i fast forward the media like remote

### DISCUSSION THREAD

**Discussion by han7794** [wizmo2/zidoo-player#37] [Ideas]
Hi wizmo,
Any possiblilities to fast forward the media like holding arrow keys on the remote controller?

**Comment by wizmo2** [comment]
The integration is limited by the available API calls (Fast-Forward, Next Media, number buttons, etc).  You could try an automation to send multiple calls with the Remote interface. (maybe a button to enable and disable a sequence of send keys and delays)  

The HA Remote interface does support a repeat/delay and hold, but I've not looked into it and it is not implemented as yet.   If the repeat/delay automation works for you, I could look at adding in a future release.

**Comment by han7794** [comment]
thanks for the reply
I tried repeat calling,it can only fast forward at the fixed frequency about 10 seconds.If the media is long,this solution will take some time.When holding the arrow buttons on the real remote controller,fast forward speed will be increased,thats what i want to see.

**Comment by wizmo2** [comment]
Just had a quick look.  The repeat and delay on the remote send keys do work.  You can set the number of repeats and the delay.  May be worth experimenting with.  For example you could try an automation that decreases the delay the longer its enabled.  It does seem to be more limited than the IR, which will be down to API latencies.  You could potentially use the numeric keys for big steps (10% steps).

Fast Forwards and Backwards may also be something you should look at.    

Beyond that, I do not see any other available API calls that could help
